### ISSUE TITLE
nuget restore fails when the source is a shared directory

### ISSUE THREAD

**Issue by TanukiSharp** [NuGet/Home#658]
I need to restore packages from the command line. When I use 

`nuget restore packages.config -PackagesDirectory ./temp -Source \\server\path\to\packages`

I get the error

`Unable to find version '<VERSION>' of package '<PACKAGE>'.`

However, when I map this path to a network drive B:\ for example, then the following

`nuget restore packages.config -PackagesDirectory ./temp -Source B:\path\to\packages`

just works perfectly.
Why can't I use `\\server\path` source ?

Mapping a drive works but is not a good solution because this is a user setting, I can't ask everybody to map a given drive in order to have it work company-wide.

Thanks in advance for your help.

**Comment by yishaigalatzer** [comment]
We think this actually does work, so it will be helpful if you can provide us with more details.

What version of nuget.exe are you using? Can you show us the actual command you are using, and the actual errors?

**Comment by TanukiSharp** [comment]
Indeed I tested with different network configurations (different smb servers, different Linux distributions, VPN or not) and there is only one case that it doesn't work but clearly unrelated with NuGet.

I guess several people will probably want to host their own feed on a Linux server, and running a NuGet server is way too complicated for the moment in my opinion, even on Windows, so once I found the issue on my side I will post it here, it may help someone.

Anyway, my apologizes the incorrect issue report.
